specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -4338,7 +4349,7 @@
 		"externalDocs": {
 			"url": "https://docs.microsoft.com/azure/search/cognitive-search-skill-language-detection"
 	    },
-		"description": "The skill detects the language of input text and reports a single language code for every document submitted on the request. The language code is paired with a score indicating the strength of the analysis."
+		"description": "A skill detects the language of input text and reports a single language code for every document submitted on the request. The language code is paired with a score indicating the confidence of the analysis."
```

Comment:
"that"

Why is it "every document submitted on the request"? Doesn't this skill take a single string as input?

Reply:
I copied it from this link: https://docs.microsoft.com/azure/search/cognitive-search-skill-language-detection. I think we can send a batch request for language detection.